concrete-core/src/backends/core/private/crypto/ggsw/ciphertext.rs

```diff
@@ -69,6 +76,29 @@ impl<Scalar> GgswCiphertext<Vec<Scalar>> {
     }
 }
 
+impl<Scalar> GgswCiphertext<Vec<Scalar>>
+where
+    Scalar: UnsignedTorus,
+{
+    pub fn new_trivial_encryption(
+        poly_size: PolynomialSize,
+        glwe_size: GlweSize,
+        decomp_level: DecompositionLevelCount,
+        decomp_base_log: DecompositionBaseLog,
+        plaintext: &Plaintext<Scalar>,
+    ) -> Self {
+        let mut ciphertext = Self::allocate(
+            Scalar::ZERO,
+            poly_size,
+            glwe_size,
+            decomp_level,
+            decomp_base_log,
+        );
+        ciphertext.fill_with_trivial_encryption(plaintext);
```

Comment:
Here I think you can drop the `;` and remove the `ciphertext` line below.

Reply:
The method on line 97 does not return anything, it only updates the internal state of ciphertext. So we need line 98 for the function to actually return something. 
(Or at least I guess so, maybe some Rust expert can confirm?)

Reply:
You are right @orelcosseron ;)